duckduckgo/iOS
# Translations

Task/Issue URL: https://app.asana.com/0/856498667320406/1176971145104443
Tech Design URL: https://app.asana.com/0/856498667320406/1186381273740755
CC:

**Description**:
Translations plus some automations to make import/export easier.

Notes:
- Still waiting for translation for few strings (Welcome to... and one dax onboarding).

**Steps to test this PR**:

1. Run the app, ensure it is localized.
2. See if layout works as expected.
3. Validate new scripts. 



**Copy Testing**:


**Orientation Testing**:


**Device Testing**:


**OS Testing**:


**Theme Testing**:


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
